cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskManager.java

```diff
@@ -22,63 +23,83 @@
 /**
  * The class that helps track the execution status for the balancing.
  * It does the following things:
- * 1. Keep track of the in progress partition movements between each pair of source-destination broker.
- * 2. When one partition movement finishes, it check the involved source and destination broker to see
- * if we can run more partition movements.
+ * 1. Keep track of the in progress partition movements between each pair of source-destination disk or broker.
+ * 2. When one partition movement finishes, it checks the involved brokers to see if we can run more partition movements.
  * We only keep track of the number of concurrent partition movements but not the sizes of the partitions.
  * Because the concurrent level determines how much impact the balancing process would have on the involved
  * brokers. And the size of partitions only affect how long the impact would last.
  *
  * The execution task manager is thread-safe.
  */
 public class ExecutionTaskManager {
-  private final Map<Integer, Integer> _inProgressReplicaMovementsByBrokerId;
+  private final Map<Integer, Integer> _inProgressInterBrokerReplicaMovementsByBrokerId;
```

Comment:
Can we move the common logic in this class to another patch in `master`? -- e.g. renames `_inProgressReplicaMovementsByBrokerId` -> `_inProgressInterBrokerReplicaMovementsByBrokerId`